### ISSUE TITLE
how to add resources for each target 

### ISSUE THREAD

**Issue by knight6700** [brightdigit/PackageDSL#3]

**Comment by leogdion** [comment]
Do you have a use case? What's your ideal API for it?
I'd probably use some sort of modifier (think SwiftUI modifier)

**Comment by knight6700** [comment]
i fixed issue moved fonts from package to app 
but if i want to add resources fro target like 
```
resources: [
                .process("Styling/Colors/Colors.xcassets"),
                .process("Styling/Fonts/FontsFiles"),
                .process("Images/Images.xcassets"),
                .process("Images/whiteLoading.json"),
                .process("Images/primaryLoading.json")
            ]
 ```

**Comment by knight6700** [comment]
@leogdion  i think we need to make a resources for each target like 
```
var resources: [PackageDescription.Resource] {
        [
            .process("../../Sources/HorizonKit/Styling/Fonts/FontsFiles")
        ]
    }
```

**Comment by leogdion** [comment]
Take a look at #5 and let me know if that's what you are looking for.

**Comment by leogdion** [comment]
Fixed in #5
